src/app/shared/auth/auth.inceptor.ts

```diff
@@ -0,0 +1,20 @@
+import { BrowserStorageService } from './../storage.service';
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpInterceptor, HttpHandler, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class TokenInterceptor implements HttpInterceptor {
+
+    constructor(private storageService: BrowserStorageService) { }
+
+    intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        const userToken = this.storageService.get('token');
+        if (userToken) {
+            const modifiedReq = req.clone({ 
+                headers: req.headers.set('Authorization', `Bearer ${userToken}`),
+            });
+            return next.handle(modifiedReq);
+        }
```

Comment:
looks like in case when `userToken` is not present this interceptor will hold this request.
```typescript
    const userToken = this.storageService.get('token');

  return next.handle(
    userToken ? 
    req.clone({  headers: req.headers.set('Authorization', `Bearer ${userToken}`) }) :
    req
);
```